### ISSUE TITLE
Routes for _index.json are appearing as /index/

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#341]
@jamestagal flagged to me that there is a bug introduced in v0.6.60 that is making `_index.json` pages appear not as true index files but under an "index" route instead, like: http://localhost:3000/index/

This was introduced because I tried to fix another bug that was only removing the `.json` extension from `:filename` route replacement patterns if the replacement pattern was at the end of the URL.

**Comment by jimafisk** [comment]
This should be fixed in [v0.6.61](https://github.com/plentico/plenti/releases/tag/v0.6.61)

Please just let me know if you continue experiencing routing issues! Thanks!

**Comment by jimafisk** [comment]
This new fix introduces new bugs with special files like `_defaults.json`:

```
Can't create props for content/audits/_defaults.json                                                                                                                                          
Can't render htmlComponent for layouts_content_audits_svelte                                                                                                                                  
TypeError: Cannot read properties of undefined (reading 'link')
```

**Comment by jimafisk** [comment]
This should be fixed now in [v0.6.62](https://github.com/plentico/plenti/releases/tag/v0.6.62)
